src/PrestaShopBundle/Resources/config/admin/services_alias.yml

```diff
@@ -2,12 +2,9 @@ services:
     # Service aliases for backward campatibility
     prestashop.core.api.stockMovement.repository:
         alias: prestashop.core.api.stock_movement.repository
-        deprecated: 'The "%service_id%" service is deprecated. Use "prestashop.core.api.stock_movement.repository" instead'
```

Comment:
Is this `deprecated` label now deprecated? Or you just don't like it? :)

Reply:
it's deprecated since Symfony 2.8 xD